### PR TITLE
Minor tweaks, updates, typos, and more ComDev resources

### PR DESCRIPTION
**Pull request by rbowen**

### CODE REVIEW

File: content/Apache/ComDev/ComDevIntro/src/main/asciidoc/index.adoc [hunk 1]
```diff
@@ -137,14 +135,35 @@ Everything is archived so you can easily go back and search.
 * How can we help support you in your open source journey?
 
 == Newcomer resources
-* Getting Started http://community.apache.org/newcomers/index.html
-* FAQs http://community.apache.org/newbiefaq.html
+* Getting Started https://community.apache.org/newcomers/index.html
+* FAQs https://community.apache.org/newbiefaq.html
 * ASF Brochure https://cwiki.apache.org/confluence/display/COMDEV/ASF+Information+Brochure
-** Help Wanted https://helpwanted.apache.org/
```
Comment (lfrancke) [inline content/Apache/ComDev/ComDevIntro/src/main/asciidoc/index.adoc:143]: Out of curiosity (this does not block this PR): What's happened to this? It's still online but...not used I assume?

**Review by lfrancke — APPROVED** [review]

File: content/Apache/ComDev/ComDevIntro/src/main/asciidoc/index.adoc [hunk 2]
```diff
@@ -137,14 +135,35 @@ Everything is archived so you can easily go back and search.
...
+** ALC Bangalore
+** ALC Beijing
+** ALC Budapest
+** ALC Hangzhou
+** ALC Indore
+** ALC Lagos
+** ALC Shanghai
+** ALC Shenzhen
+** ALC Warsaw
+** ALC Xi'an
```
Comment (lfrancke) [inline content/Apache/ComDev/ComDevIntro/src/main/asciidoc/index.adoc]: When rendering the presentation this one drops off the bottom for me. I guess it depends on your resolution etc. and I'm happy to merge as is but maybe we can somehow create a two column table instead or something similar?

Comment (rbowen) [reply content/Apache/ComDev/ComDevIntro/src/main/asciidoc/index.adoc]: Sure. I think I can probably figure out how to do that, but if someone else already knows, that would be great.

Comment (lfrancke) [reply content/Apache/ComDev/ComDevIntro/src/main/asciidoc/index.adoc]: I don't, sorry :(

I'm happy to merge this as is as well. I'll leave the decision up to you.

Comment (justinmclean) [reply content/Apache/ComDev/ComDevIntro/src/main/asciidoc/index.adoc]: Take a look at https://github.com/apache/incubator-training/blob/develop/content/Conferences/ApacheCon/src/main/asciidoc/index_en.adoc I have an example of 2 columns in that

Comment (justinmclean) [reply content/Apache/ComDev/ComDevIntro/src/main/asciidoc/index.adoc]: You'll need to include a small bit of CSS as well to get that to work

File: content/Apache/ComDev/ComDevIntro/src/main/asciidoc/index.adoc (hunk 1 above)
Comment (rbowen) [reply content/Apache/ComDev/ComDevIntro/src/main/asciidoc/index.adoc:143]: The content in there is outdated to the point of being useless. If someone wanted to take it on, they would need to evangelize it to projects and get them to use it in a way that would actually result in useful contributor engagement - which never happened - and then actually respond when contributors showed up to volunteer - which also never happened.

File: content/Apache/ComDev/ComDevIntro/src/main/asciidoc/index.adoc (hunk 2 above)
Comment (rbowen) [reply content/Apache/ComDev/ComDevIntro/src/main/asciidoc/index.adoc]: I have attempted to get this to work, copying your example exactly, but cannot get it to render in columns. Perhaps you could just merge, and someone else could tweak once it's there? Is that a possibility?

**Pull request merged**